contracts/Lens/PoolLens.sol

```diff
@@ -302,4 +302,179 @@ contract PoolLens is ExponentialNoError {
                 underlyingPrice: priceOracle.getUnderlyingPrice(address(vToken))
             });
     }
+
+    /**
+     * @dev Struct for Pending Rewards for per market
+     */
+    struct PendingReward {
+        address vTokenAddress;
+        uint256 amount;
+    }
+
+    /**
+     * @dev Struct for Reward of a single reward token.
+     */
+    struct Reward {
+        address distributorAddress;
+        address rewardTokenAddress;
+        uint256 totalRewards;
+        PendingReward[] pendingRewards;
+    }
+
+    /**
+     * @dev Struct used in RewardDistributor to save last updated market state
+     */
+    struct RewardToken {
```

Comment:
The name of this struct is a bit unclear based on what it contains. Maybe `RewardDistributionIndex`?

Reply:
I just copied the struct used in RewardsDistributor. Sure I will change that

Reply:
yeah I didn't catch that name but I have the same comment.

you can even see it is labeled as state in the variable here 
```mapping(address => RewardToken) public rewardTokenSupplyState;```

Reply:
Changes it to `RewardTokenState`